src/libs/compound-v3/utils.test.ts

```diff
@@ -1,37 +1,46 @@
 import { BigNumber } from 'ethers';
 import BigNumberJS from 'bignumber.js';
-import { calcAPR, calcHealthRate, calcNetAPR, calcUtilization, getMarketLabel } from './utils';
+import { calcAPR, calcHealthRate, calcNetAPR, calcUtilization, transformMarketId } from './utils';
 import * as common from '@protocolink/common';
 import { expect } from 'chai';
-import * as logics from '@protocolink/logics';
 
-describe('Test getMarketLabel', function () {
+describe('Test transformMarketId', function () {
   const testCases = [
     {
       chainId: common.ChainId.mainnet,
-      marketId: logics.compoundv3.MarketId.USDC,
+      marketId: 'usdc',
```

Comment:
Curious about the benefits of using `usdc` instead of using `logics.compoundv3.MarketId.USDC`

Reply:
The reason for using `usdc` instead of `logics.compoundv3.MarketId.USDC` is primarily for testing purposes to simulate real-world API scenarios.
In our API, the `marketId` is typically provided in lowercase, so using `usdc` directly allows us to closely mimic the API behavior in our tests.